### ISSUE TITLE
Is there a way to make it work only in windows (not games) or quick profile toggle?

### ISSUE THREAD

**Issue by MiChAeLoKGB** [Ryochan7/antimicro#288]
As title says,

is there a way to quickly toggle profiles (maybe a button on controller that I could map to switch between profiles? I know I can set up app that will make it switch to profile, but I would like my profile (made to control windows and KMPlayer from couch) to be active only when I am not in game (as I want it in full screen apps, IDK how it would work detecting full screen movie).

I know my explanation is kinda lackluster, but I was never good at it.

Basically:
Set up a button on controller (combination of buttons would be best), that would switch/toggle my profile, so I can quickly disable it when Im opening Steam big picture/games.

Ps.: I think button might be easiest way to do it. Automation would be hard to really get working properly.

Thanks.

**Comment by Ryochan7** [comment]
There is a way to have the program switch profiles via a button press. You would have to use a Load slot in an assignment and specify the location of the profile that you want loaded. There is no support for direct button combinations so you would have to use sets in order to allow a similar assignment to happen via a button combination.

**Comment by MiChAeLoKGB** [comment]
Yes, I managed ti find that out, but as its not a combination of buttons that does this, I am positive you would switch those profiles multiple times while in game, which is not the best idea.

Is there any way this could be added?
ie.: I could set up to push RT + LT + BACK (or any other combo) to switch to profile.

Or maybe having a "modifier" button like keyboards do with CTRL & ALT, so you would set one button to be modifier and when you hold it, other button presses would do nothing unless they are part of combo with that modifier (so you could have double the fun with infinite combo options with modifier)).

I hope you understand what I mean.

Ps.: feel free to explain challenges this would introduce, as programmer I could maybe help a bit by finding correct way to do it if possible.
